definition/data/sheets/CaseField.json

```diff
@@ -90,5 +90,13 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "directionOrder", "Label": "Direction order", "FieldType": "DirectionOrder", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "channel", "Label": "Channel", "FieldType": "FixedList", "FieldTypeParameter": "ChannelType", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "intentionToProceedDeadline", "Label": "Intention To Proceed Deadline", "FieldType": "Date", "FieldTypeParameter": "ChannelType", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "state", "Label": "state", "FieldType": "Text", "SecurityClassification": "Public"}
+  {"CaseTypeID": "MoneyClaimCase", "ID": "state", "Label": "state", "FieldType": "Text", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "issueLetterContact", "Label": "Who is the letter to?", "FieldType": "FixedRadioList", "FieldTypeParameter": "ChangeContactDetailsParty", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "letterContentHeader", "Label": "## Enter the text of the letter", "FieldType": "Label", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "letterContentPrefixClaimant", "Label": "Dear ${applicants.partyName},", "FieldType": "Label", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "letterContentPrefixDefendant", "Label": "Dear ${respondents.partyName},", "FieldType": "Label", "SecurityClassification": "Public"},
```

Comment:
This doesn't look like a label. Is this correct? (same for line above)

Reply:
same here